static/app/views/insights/common/views/spans/selectors/actionSelector.tsx

```diff
@@ -43,12 +43,15 @@ export function ActionSelector({value = '', moduleName, spanCategory, filters}:
 
   const useHTTPActions = moduleName === ModuleName.HTTP;
 
-  const {data: actions} = useSpansQuery<Array<{'span.action': string}>>({
-    eventView,
-    initialData: [],
-    enabled: !useHTTPActions,
-    referrer: 'api.starfish.get-span-actions',
-  });
+  const {data: actions} = useSpanMetrics(
+    {
+      search: eventView.query,
```

Comment:
This is so close to just dumping that annoying `EventView` creation! Could you replace this with calling `buildEventViewQuery` directly?